Patches/Rimsenal Collection/Enhanced Vanilla/Weapons_Charge_CE.xml

```diff
@@ -156,6 +155,7 @@
 					<ammoSet>AmmoSet_12GaugeCharged</ammoSet>
 				</AmmoUser>
 				<FireModes>
+					<aiAimMode>Snapshot</aiAimMode>
```

Comment:
Fine, but techincally not needed as aiAimMode Defaults to Snapshot. Handy to have the field for visibilty, but yeah